jsk_spot_robot/jsk_spot_startup/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_spot_startup</name>
+  <version>0.0.0</version>
```

Comment:
change to 1.1.0,

add author section

Reply:
update the package.xml
- changed the version to 1.1.0
- changed the maintainer
- added an author section
- added dependencies